cinema_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
-)
+SECRET_KEY = ("django-insecure-ru6ndfka@95_"
+              "(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy")
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`, which is not recommended for production environments as it can expose sensitive information. Ensure that `DEBUG` is set to `False` in production.